docs/resources/aws_flow_log.md.erb

```diff
@@ -0,0 +1,82 @@
+---
+title: About the aws_flow_log Resource
+platform: aws
+---
+
+# aws\_flow\_low
+
+Use the `aws_flow_log` InSpec audit resource to test properties of a single Flow Log.
+
+## Syntax
+
+    describe aws_flow_log('fl-9c718cf5') do
+      it { should exist }
+    end
+
+## Resource Parameteres
+### flow_log_id
+
+This resource expects a single parameter that uniquely identifies the Flow Log. You may pass it as a string, or as the value in a hash:
```

Comment:
Because you can also search by subnet or VPC ID, I think we might want to soften "expects a single parameter" to something like "accepts a single parameter or other search terms"